server/problems/svg-creator.js

```diff
@@ -0,0 +1,39 @@
+/** @exports */
+const newImage = (sourceImage = '<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" version="1.1"></svg>') => ({
```

Comment:
Очень большая строка
```suggestion
const defaultSvgImage = '<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" version="1.1"></svg>';

const newImage = (sourceImage = defaultSvgImage) => ({
```